packages/platform/setup-workqueue.js

```diff
@@ -20,14 +19,9 @@ export default ({ cronText }) => {
       WorkerDirector,
     })
   );
-  // handlers.push(
-  //   new IntervalWorker({
-  //     WorkerDirector,
-  //   })
-  // );
   if (cronText) {
     handlers.push(
-      new CronWorker({
+      new IntervalWorker({
         WorkerDirector,
         cronText,
```

Comment:
Has the format change, is it Linux corn text now.

Reply:
No I think it's the same like before, isn't it?

Reply:
You're right it's actually not cron it's kind of a text parser (http://bunkat.github.io/later/parsers.html#text) so the name is missleading, we should support both

Reply:
I saw we have this problem at different places there is even a setting "autoSchedulingCronText" that is actually not cron it's later.js text expression.